logging.go

```diff
@@ -23,6 +24,13 @@ func LogToFile(path string, prefix string) (*os.File, error) {
 		return nil, err
 	}
 	log.SetOutput(f)
+
+	// Add a space after the prefix if one doesn't exist
+	finalPrefixChar := prefix[len(prefix)-1]
+	if len(prefix) > 0 && !unicode.IsSpace(rune(finalPrefixChar)) {
```

Comment:
The `len` check should probably be dealt with before accesing `prefx[len(prefix)-1]`.

Reply:
Oh, you’re right. Thanks @muesli.